server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/AliasMappingTest.java

```diff
@@ -142,15 +147,51 @@ public void messageShouldRedirectToUserWhenSentToHisAlias() throws Exception {
     }
 
     @Test
-    public void messageShouldRedirectToUserWhenSentToHisAliasWithUpperCase() throws Exception {
-        webAdminApi.put(AliasRoutes.ROOT_PATH + "/" + BOB_ADDRESS + "/sources/" + BOB_ALIAS);
+    public void messageShouldRedirectToUserWhenSentToHisUpperCaseAlias() throws Exception {
+        webAdminApi.put(AliasRoutes.ROOT_PATH + "/" + BOB_ADDRESS + "/sources/" + UPPER_CASE_ALIAS.toLowerCase());
+
+        messageSender.connect(LOCALHOST_IP, jamesServer.getProbe(SmtpGuiceProbe.class).getSmtpPort())
+            .sendMessage(FakeMail.builder()
+                .name("name")
+                .mimeMessage(message)
+                .sender(ALICE_ADDRESS)
+                .recipient(UPPER_CASE_ALIAS));
+
+        imapMessageReader.connect(LOCALHOST_IP, jamesServer.getProbe(ImapGuiceProbe.class).getImapPort())
+            .login(BOB_ADDRESS, PASSWORD)
+            .select(IMAPMessageReader.INBOX)
+            .awaitMessage(awaitAtMostOneMinute);
+        assertThat(imapMessageReader.readFirstMessage()).contains(MESSAGE_CONTENT);
+    }
+
+    @Test
+    public void messageShouldRedirectToUpperCaseDefinedUserWhenSentToHisLowerCaseAlias() throws Exception {
```

Comment:
I was more expecting messageShouldRedirectToUpperCaseUserWhenSentToHisLowerCaseAlias (ie without the "Defined"), because defining an uppercase user is the same thing than defining a lowercase one.

Reply:
yes but... we send to the lowercase alias anyway? Not sure to exactly get what you expect with this test then? Or you are just suggesting a better name for the test?

Reply:
yes and no. The test would be the same, just the user would not be created before.

I expect this test to check that the creation of an alias is the same if the alias point to an upper case user or to a lower cased user.

Reply:
But if you send a message to the alias of a user that has not been created then... logically you cannot check that the message has been received because... well he doesn't exist?

Reply:
Ah no I think I got it... like the mapping is to the user in upper case, like `BOB`, while the user has been created like `bob`, is that it?

Reply:
yes that's it :)